modules/popup/react/lib/focus-trap-js.js

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable */
+// refactor for v5
```

Comment:
This is the `focus-trap-js` library inlined into v4 with changes included from https://github.com/alexandrzavalii/focus-trap-js/pull/11

Reply:
v5 introduces `useFocusRedirect` which hard-codes similar logic from this library, but without the hard-coded focus trap logic. This should all be refactored for v5